test/index.ts

```diff
@@ -146,20 +238,35 @@ function processEnvVars(): {
     shouldRunTests: boolean;
     shouldRunTypeChecks: boolean;
     shouldRecordBenchmarks: boolean;
+    shouldFuzz: boolean;
 } {
     const runTests = process.env.AZLE_RUN_TESTS ?? 'true';
     const runTypeChecks = process.env.AZLE_RUN_TYPE_CHECKS ?? 'true';
     const recordBenchmarks = process.env.AZLE_RECORD_BENCHMARKS ?? 'false';
+    const fuzz = process.env.AZLE_FUZZ ?? 'false';
```

Comment:
Can we rename this to runFuzzTests and AZLE_RUN_FUZZ_TESTS?

Reply:
I like it how it is, we discussed this a bit, we can discuss more.

Reply:
I'll leave further discussion for when we tackle https://github.com/demergent-labs/azle/issues/2286